wiremock/scripts/stubApis.ts

```diff
@@ -1,11 +1,88 @@
 /* eslint-disable no-console */
 import { apiPaths } from '../../server/paths'
 import { stubFor } from '../index'
+import organizations from '../linkage/organizations.json'
 import { courseOfferings, courseParticipations, courses, prisoners, referrals } from '../stubs'
+import referralSummaries from '../stubs/referralSummaries.json'
```

Comment:
Currently getting 404ing pages when clicking on links in the table, is that to be expected? Looks like we're not stubbing referral views here with your new referral summaries - do we need to add matching referrals to `referrals.json` for this, or is this PR just to get a basic page loading?

Reply:
I only went so far with it to make the summary page load, the next step would be this.